### ISSUE TITLE
Refactor current-projects-check file to fix any formatting issue with new tools data format

### ISSUE THREAD

**Issue by ronaldpaek** [hackforla/website#4861]
### Overview

The purpose of this issue is to refactor the code in the current-project-check file to handle a list of strings(tools), instead of a single one. This change will maintain consistent formatting and resolve problems introduced by the recent change in data format.

### Action Items

- [x] Refer to [current-projects-check.js](https://github.com/hackforla/website/blob/gh-pages/assets/js/current-projects-check.js) and locate where tools is being referenced
- [x] Refactor this code so tools can work as a comma separated String or an Array
```
{%- if project.tools -%},
"tools": `{{ project.tools }}`
{%- endif -%}
```
to:
```
{%- if project.tools -%},
"tools": {{ project.tools | jsonify }}
{%- endif -%}
```

- [x] Refactor this code to map over tools since tools will be an Array now and not a String

```
${project.tools ?
`
<div class="project-tools">
<strong>Tools: </strong>
${ project.tools }
</div>
`:""
}
```
to:
```
${project.tools ?
`
<div class="project-tools">
<strong>Tools: </strong>
${(Array.isArray(project.tools) ? project.tools : project.tools.split(','))
    .map(tool => `<p class='project-card-field-inline'> ${tool}</p>`)
    .join(", ")
}
</div>
`: ""
}
```

- [x] After completing both changes this should allow us to make changes to tools without effecting the functionality of the website

Merge team:
- [x] Once this issue is merged, release its dependency in #4777
### Resources/Instructions

- [Projects](https://www.hackforla.org/projects/) page
- [Liquid template language](https://shopify.github.io/liquid/)
-  This issue is part of an epic: #4427

**Comment by AlimurtuzaCodes** [comment]
Can I work on this issue?

**Comment by ronaldpaek** [comment]
@t-will-gillis can they start working on this? I think it'll be okay since it should output same result, but you can respond to this thread.

**Comment by t-will-gillis** [comment]
Hi @CYBWEBALI - thank you for your interest! Before working on an issue, you need to be a member of Hack for LA. If you have not joined yet, please follow the steps on our [Getting Started](https://www.hackforla.org/getting-started) page.

**Comment by ExperimentsInHonesty** [comment]
I just left a message on #4860 and that applies here too

https://github.com/hackforla/website/issues/4860#issuecomment-1619312877

> Hi @ronaldpaek, what is the state of each of the images you have provided. Is one the current state and the second the future state? And if so, the second one is not showing spaces between, and that doesn't look good. It was my understanding that this change was just to made it possible to do the projects check page easier using the standards of the technologies field. Not sure why there would be any visual change.

**Comment by ronaldpaek** [comment]
@ExperimentsInHonesty, I see, yes those images were of the two state of the data in as one long string with commas or in a list of strings. I wanted to verify that the code would not break and cause the an error. 

I think we should change all the data to list format first then, but once I think either way there will be some minor spacing issues with the commas till all the revisions are done, but the code shouldn't break though.

**Comment by ronaldpaek** [comment]
@ExperimentsInHonesty I can remove those photos if you want, I know it was confusing.

**Comment by t-will-gillis** [comment]
@ExperimentsInHonesty please see my note on #4860. I confirmed that the @ronaldpaek 's code for the project Tools: should work before, during, and after the transition from strings to list/array datatypes. Of course, during and after the transition period we can monitor to make sure there are no glitches.

**Comment by ronaldpaek** [comment]
@roslynwythe I guess this is complete, right? Sorry I was working on another issue.
